src/adapters/supabase/helpers/client.ts

```diff
@@ -5,6 +5,13 @@ import { Database } from "../types";
 import { InsertPermit, Permit } from "../../../helpers";
 import { BigNumber, BigNumberish } from "ethers";
 
+interface accessLevels {
```

Comment:
```suggestion
interface AccessLevels {
```
be mindful about naming conventions